multicluster/config/overlays/leader-ns/kustomizeconfig.yaml

```diff
@@ -0,0 +1,53 @@
+nameReference:
+  # Since mutating webhook configuration name gets prefixed during kustomization,
+  # propagate the actual names into the corresponding cluster-role that is
+  # necessary for mcs-controller to have access to those webhook configuration.
```

Comment:
s/configuration/configurations/

Reply:
btw, the description in issue summary needs update.

Reply:
I did update commit message, dont know why it doesnt show up. Ill try to look into it